Homework6/MapFilterFold/MapFilterFold/Functions.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace MapFilterFold;
+
+using System.Collections.Generic;
+
+/// <summary>
+/// Static class for static methods Map, Filter and Fold.
+/// </summary>
+public static class MapFilterFold
+{
+    /// <summary>
+    /// Applies function to all elements of the list.
+    /// </summary>
+    /// <param name="list">Input list.</param>
+    /// <param name="function">Function that will be applied to elements.</param>
+    /// <typeparam name="T">Type of elements in input list.</typeparam>
+    /// <typeparam name="R">Type of output value of the function.</typeparam>
+    /// <returns>List of elements to which the function has been applied</returns>
+    public static List<R> Map<T, R>(List<T> list, Func<T, R> function)
```

Comment:
```suggestion
    public static List<R> Map<TIn, TOut>(List<TIn> list, Func<TIn, TOut> function)
```
Имя типа должно с большой буквы `T` начинаться.